.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -48,7 +48,7 @@ body:
     attributes:
       label: Relevant logs
       description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
-      render: shell
+      render: Shell
```

Comment:
GH uses lowercase in their examples: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

Reply:
Shame on them - it doesn't pass their own schema validation 😄 
https://json.schemastore.org/github-issue-forms.json